### ISSUE TITLE
Is my account calibration bugged forever?

### ISSUE THREAD

**Issue by simplyexperimental** [ValveSoftware/Dota2-Gameplay#15597]
### Description

TLDR: If I win more than 50% of my games during calibration how is it possible I lose 500-1000 MMR upon calibrations end?
This happens to me and my friend but not our other friend EACH TIME we calibrated over the years. Can something be done about this?
=============
LONG: I have only ever had one account since I joined Dota in 2016 - however I've noticed other users who calibrate get lower mmr while some do not despite performing the same.

As far as I am aware if you win more games then you lose you will gain more mmr than when you calibrated right?
So if I had 3000 MMR and then entered calibration and then won 7/10 games that should certainly mean you will be calibrate to higher than 3000 MMR right?

I am asking this because me and my friend WITHOUT FAIL will ALWAYS LOSE MMR every time we calibrate even if we win more than half the games out of 10. In fact we lose about 500-1000 mmr each time we calibrate.

Since this has been consistent with our accounts I wonder if this problem is an account specific issue and what we can do about it?

**Comment by Memoframe** [comment]
I was almost 5k but it was by spamming one hero support.
Then I did not play ranked for a long time.
When they removed rank if you did not recalibrate I calibrated again.
Drop was like 1k+, to 3800 or something.
Tbh it's my "real" mmr, I mean I feel like that's where I belong.
But over time I climbed to 4400 iirc and definitely can climb more, just because I play as support, and it's easier for me to climb when someone else does the job :D my micro sux, I'm bad, have no other way. 
Imo it's correct to flatten everyone, if you are a certain rank you'll climb back.

Most likely it looks at a list of parameters, not only win/loss. For example KDA, GPM, LH/D, total damage, fight participation etc.

**Comment by simplyexperimental** [comment]
> I was almost 5k but it was by spamming one hero support. Then I did not play ranked for a long time. When they removed rank if you did not recalibrate I calibrated again. Drop was like 1k+, to 3800 or something. Tbh it's my "real" mmr, I mean I feel like that's where I belong. But over time I climbed to 4400 iirc and definitely can climb more, just because I play as support, and it's easier for me to climb when someone else does the job :D my micro sux, I'm bad, have no other way. Imo it's correct to flatten everyone, if you are a certain rank you'll climb back.
> 
> Most likely it looks at a list of parameters, not only win/loss. For example KDA, GPM, LH/D, total damage, fight participation etc
**I would like a developer to look into this.**
I understand there might be more to how calibration works - but I think I've calibrated 6 times over the years and each and every time I win usually 5-7 out of 10 games - and in each case I lose 500 MMR on average - this must be a bug because I heard every win is worth 50 mmr during calibration. 

Unless of course wins and losses don't actually mean anything and the AI is looking for key behaviours only higher and low players do and using that to mark the rank. For example I would think that if in those 10 games you never buy dust or smoke the AI would consider that a huge red flat and rank u down who knows...

**Comment by anko** [comment]
It seems to me that calibration works based on something other than win percentage. It's not necessary to look at win percentage to judge someone's skill.

For example, the calibration algorithm could put you in matches that you are statistically likely or unlikely to win, and see if its predictions come true. Then base the MMR change on how right its predictions were. It doesn't need to give you evenly matched games to be able to do that. In such a system, you can win very few games and still gain MMR (if it gave you statistically unlikely games that you managed to win some of anyway), or win almost all of them and still lose MMR (if it gave you statistically easy games but you still lost a few of them).

**Comment by commentator24** [comment]
i'm not sure if you guys pay attention to what role you've been playing but you wont calibrate with more mmr if you keep changing role. pos 4 is not the same as pos 5 hidden rating.
e.g. i only play pos 1 few times a year. pick phase will say crusader 5 even though medal is archon 3. 
this means if i play pos 1 for my final calibration i would lose a lot of mmr even if i win a lot. thats how u would lose a lot of mmr during calibration. basically win 7/10 as pos 1 might calibrate archon 1 instead of other roles keep losing but still get archon 2/3/4. 

check profile rating before calibrating and sort by rating.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/e243a697-cf1e-426b-8440-194416dbf70e)
as you can see here my pos 1 seems to have highest rating, but actually the lowest mmr, because not enough total games i guess.
once you calibrate you should play each role once to kind of update the hidden rating number. i already play each role until the bar kinda looks similar. 

safest to stick with the role with most total games if you want back old mmr.
e.g. only change role if start to lose 3x in a row. next best option maybe highest rating. if enemy average medal seems lower then don't continue that role and try different one.
